tests/RepositoryForms/Data/Mapper/ContentTypeDraftMapperTest.php

```diff
@@ -162,6 +164,30 @@ public function testMapToFormData()
         ]);
         $expectedContentTypeData->addFieldDefinitionData($expectedFieldDefData2);
 
-        self::assertEquals($expectedContentTypeData, (new ContentTypeDraftMapper())->mapToFormData($contentTypeDraft));
+        $eventDispatcherMock = $this->getMockBuilder(EventDispatcherInterface::class)->getMock();
```

Comment:
AFAIR there's a syntax sugar there:
```suggestion
        $eventDispatcherMock = $this->createMock(EventDispatcherInterface::class);
```